pwndbg/aglib/disasm/disassembly.py

```diff
@@ -98,11 +99,8 @@ def clear_on_reg_mem_change() -> None:
 
 
 @pwndbg.lib.cache.cache_until("objfile")
-def get_disassembler(address: int, cs_info: Tuple[int, int] = None):
-    if cs_info is not None:
-        arch, mode = cs_info
-    else:
-        arch, mode = pwndbg.aglib.arch.get_capstone_constants(address)
+def get_disassembler(cs_info: Tuple[int, int]):
+    arch, mode = cs_info
```

Comment:
Random comment: since we unpack it anyway, maybe its better to just accept/pass `arch, mode`? Idk, just saying, maybe its better as it is.